rs/scripts/notes.md

```diff
@@ -1,5 +1,8 @@
 // Run this in openchat repo to find public key
 dfx canister call user_index public_key '(record { })'
 
+// Run this to create a canister
+dfx canister create greet_bot --no-wallet
+
 // Bot endpoint looks like this (replace with your canister id)
-http://gc5gl-leaaa-aaaaa-qaara-cai.localhost:8080/
+http://gf4a7-g4aaa-aaaaa-qaarq-cai.localhost:8080
```

Comment:
Is this file meant to be in source control? You added notes.md to .gitignore.

Reply:
I added notes.md at the root which is ignored. Don't worry about this for now. When I write up the docs this will disappear - until then it doesn't do any harm.